src/helpers.cpp

```diff
@@ -115,8 +115,11 @@ ada_really_inline bool shorten_path(std::string& path,
 
   // Remove path’s last item, if any.
   if (!path.empty()) {
-    path.erase(path.rfind('/'));
```

Comment:
we can remove path not empty if statement. rfind already handles the case where it is empty.

Reply:
Good point.